resources/[system]/scoreboard/ui/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import { ScoreboardProvider } from "./context/ScoreboardContext";
+
+ReactDOM.render(
+  <React.StrictMode>
+    <ScoreboardProvider>
+      <App />
+    </ScoreboardProvider>
+  </React.StrictMode>,
+  document.getElementById('root')
+);
+
+// If you want to start measuring performance in your app, pass a function
```

Comment:
Do we?

Reply:
Well, good old CRA comment. Can delete it.